### PR TITLE
feat: auth-user-transactions

### PR DESCRIPTION
**Pull request by mrdrozdy**

### CODE REVIEW

File: backend/src/main/java/pl/pollub/backend/expenses/ExpenseService.java [hunk 1]
```diff
@@ -29,12 +30,12 @@ public Expense updateExpense(Long id, Expense updatedExpense) {
                 })
                 .orElseThrow(() -> new RuntimeException("Nie znaleziono wydatku o podanym identyfikatorze: " + id));
```
Comment (atakowiec) [inline backend/src/main/java/pl/pollub/backend/expenses/ExpenseService.java:31]: change each RuntimeException to HttpException with suitable http status code (404 in this case)

File: backend/src/main/java/pl/pollub/backend/incomes/IncomeRepository.java [hunk 2]
```diff
@@ -1,6 +1,11 @@
 package pl.pollub.backend.incomes;
 
 import org.springframework.data.jpa.repository.JpaRepository;
+import pl.pollub.backend.auth.user.User;
+import java.util.List;
 
 public interface IncomeRepository extends JpaRepository<Income, Long> {
+    List<Income> findByUser(User user);
+    boolean existsByIdAndUser(Long id, User user);
+    java.util.Optional<Income> findByIdAndUser(Long id, User user);
```
Comment (atakowiec) [inline backend/src/main/java/pl/pollub/backend/incomes/IncomeRepository.java:10]: try to avoid this, instead just import it, we want code to be clean

**Review by atakowiec — CHANGES_REQUESTED** [review]
first of all use ctr + alt + l in every changed file, change RuntimeExceptions to HttpException and don't import things inline like java.util.Optional in repositories

**Review by atakowiec — APPROVED** [review]
very cool 
image

**Comment by mrdrozdy** [comment]
> very cool

Thanks dude

**Pull request merged**